solr-schema/src/main/resources/solr/conf/replication-solr-managed-schema.xml

```diff
@@ -176,6 +176,12 @@
     <field name="last_metadata_modified" type="pdate" indexed="true" stored="false" required="false"
            multiValued="false" />
 
+    <!-- site_index fields -->
+    <field name="modified_since" type="pdate" indexed="false" stored="true" required="false"
+      multiValued="false"/>
+    <field name="filter_id" type="string" indexed="true" stored="true" required="false"
```

Comment:
❓ Should `required` be set to `true`?

Reply:
We cannot right now because this schema applies to all collections so then every type would require this field.

Reply:
good to know.